Use elasticsearch_data nodes in Logstash output

Infra-Red opened the pull request:
Hey! This PR changes the default hosts configuration in logstash elasticsearch output. Instead of collocating ElasticSearch and Logstash on the same VM we can point Logstash output plugin to the DNS name of ElasticSearch data nodes. This will decrease the amount of RAM used by Ingestor instances, check the graph below:

![Ingestor Memory Usage](https://i.imgur.com/UOkKKLL.png)

I will also create a separate PR which will allow to configure Elasticsearch coordinating nodes in a high load environments.

/cc @pommi @diogomatsubara

Infra-Red committed:
Use elasticsearch_data nodes in Logstash output

Reduce RAM usage on ingestor instances

axelaris commented:
From my point of view - having a local ES drain for Logstash was a great idea, and keeps cluster configuration simpler. To resolve memory issues, we can just increase amount of memory for instance, and maybe change heap calculating formula [here](https://github.com/cloudfoundry-community/logsearch-boshrelease/blob/develop/jobs/elasticsearch/templates/bin/elasticsearch_ctl#L16).
Anyway, I'd prefer to keep this conversation for a couple of weeks to hear from other active Logsearch users.

Infra-Red commented:
Hi @axelaris ! I changed the PR and now by default Logstash will forward to local ES host. I've added link support to `ingestor_syslog` job and now operator can optionally configure forwarding to remove ES host. I've also added ops file for this `ingestor-forward-to-elasticsearch-data.yml `. Is it works for you?

Infra-Red committed:
Nil elasticsearch link for cluster_monitor

Infra-Red committed:
Remove unused link

Infra-Red committed:
Nil elasticsearch link by default

axelaris commented:
Merged.
Thank you @Infra-Red!
